docs/documentation/securing_apps/topics/oidc/supported-grant-types.adoc

```diff
@@ -0,0 +1,105 @@
+=== Supported Grant Types
+
+This section describes the different grant types available to relaying parties.
+
+==== Authorization code
+
+The Authorization Code flow redirects the user agent to {project_name}. Once the user has successfully authenticated with {project_name} an
+Authorization Code is created and the user agent is redirected back to the application. The application then uses the authorization code along with its
+credentials to obtain an Access Token, Refresh Token and ID Token from {project_name}.
+
+The flow is targeted towards web applications, but is also recommended for native applications, including mobile applications, where it is possible to embed
+a user agent.
+
+For more details refer to the https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code Flow] in the OpenID Connect specification.
+
+==== Implicit
+
+The Implicit flow redirects works similarly to the Authorization Code flow, but instead of returning an Authorization Code the Access Token and ID Token is
```

Comment:
```suggestion
The Implicit flow works similarly to the Authorization Code flow, but instead of returning an Authorization Code the Access Token and ID Token is
```